cdap-data-fabric/src/main/java/co/cask/cdap/data/stream/service/heartbeat/NotificationHeartbeatPublisher.java

```diff
@@ -37,7 +37,7 @@
   public NotificationHeartbeatPublisher(NotificationService notificationService) {
     this.notificationService = notificationService;
     this.heartbeatFeed = new Id.NotificationFeed.Builder()
-      .setNamespaceId(Constants.DEFAULT_NAMESPACE)
+      .setNamespaceId(Constants.SYSTEM_NAMESPACE)
```

Comment:
No nvm. We should standardize the ``Id`` class before it gets too ugly. Can we track it?